fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/scalar/ImplicitCastRule.java

```diff
@@ -53,7 +53,10 @@ public ScalarOperator visitCall(CallOperator call, ScalarOperatorRewriteContext
                 }
             }
         } else {
-            if (fn.functionName().equals(FunctionSet.ARRAY_MAP)) { // array_map does not need to implicit cast.
+            // functions with various data types do not need to implicit cast, such as following functions.
+            if (fn.functionName().equals(FunctionSet.ARRAY_MAP) ||
+                    fn.functionName().equals(FunctionSet.EXCHANGE_BYTES) ||
```

Comment:
why disable implicit cast for these new exchange functions?

Reply:
these functions accept any data type parameter, it is no need to cast to a destnation type.